lerna.json

```diff
@@ -2,7 +2,9 @@
   "version": "2.13.30",
   "npmClient": "yarn",
   "packages": [
-    "packages/*"
+    "packages/*",
+    "!packages/account-portal",
+    "packages/account-portal/packages/*"
```

Comment:
For what it's worth, I think this is absolutely the way to go about this. I don't think it's hacky at all. Good find.